Source/Core/VideoBackends/D3D/D3DState.cpp

```diff
@@ -389,6 +389,8 @@ ID3D11BlendState* StateCache::Get(BlendingState state)
     }
   }
 
+  const bool use_dual_source = state.usedualsrc;
```

Comment:
I should have brought this up during my initial review, but why place this here, rather than closer to its usage site?

Reply:
Thanks, that's a good idea.  That was an oversight on my part.  I've been distracted with other things.